src/components/search-input/search-input.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "./search-input.scss";
+import { FaSearch } from "react-icons/fa";
```

Comment:
I would avoid importing the `react-icons` and instead use the SVG from Figma. We could create a new `svg` file with the snippet below, and use it [this way](https://github.com/code4romania/taskforce-fe-components/blob/dan/%234_add-list-components/src/components/ListItem/ListItem.js#L5). To have it working in-line, we need [this package](https://github.com/code4romania/taskforce-fe-components/blob/dan/%234_add-list-components/package.json#L56). Please let me know if this makes sense
```svg
<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
   <path d="M12.5 11H11.71L11.43 10.73C12.41 9.59 13 8.11 13 6.5C13 2.91 10.09 0 6.5 0C2.91 0 0 2.91 0 6.5C0 10.09 2.91 13 6.5 13C8.11 13 9.59 12.41 10.73 11.43L11 11.71V12.5L16 17.49L17.49 16L12.5 11ZM6.5 11C4.01 11 2 8.99 2 6.5C2 4.01 4.01 2 6.5 2C8.99 2 11 4.01 11 6.5C11 8.99 8.99 11 6.5 11Z" fill="black"/>
</svg>
```

Reply:
Do we really need to use it inline? Can't we just `<img src={SearchIcon}>`?

Reply:
On yeah, in that case you can use it that way. No need for the inline stuff. In my example I had to change the color of the svg, so I was overthinking 😓

Reply:
Hmm, it actually breaks the build. Can you please check the logs [here](https://zeit.co/code4ro/taskforce-fe-components/ivtq509bd)?